core/templates/domain/collection/collection-node.model.ts

```diff
@@ -46,7 +46,30 @@ export class CollectionNode {
   static createFromExplorationId(explorationId: string): CollectionNode {
     return CollectionNode.create({
       exploration_id: explorationId,
-      exploration_summary: null
+      exploration_summary: {
+        category: '',
+        community_owned: false,
+        activity_type: '',
+        last_updated_msec: 0,
+        ratings: {
+          1: 0,
+          2: 0,
+          3: 0,
+          4: 0,
+          5: 0,
+        },
+        id: '',
+        created_on_msec: 0,
+        human_readable_contributors_summary: {},
+        language_code: '',
+        num_views: 0,
+        objective: '',
+        status: '',
+        tags: [''],
+        thumbnail_bg_color: '',
+        thumbnail_icon_url: '',
+        title: '',
+      }
```

Comment:
Why we need to do this?

Reply:
Because property `exploration_summary` is of type `LearnerExplorationSummaryBackendDict`. When we pass it to `create ` method we have to assign some value to it ,we can't simply assign `null` to it like we were doing before.